pyload/requests/curl/chunk.py

```diff
@@ -145,7 +147,7 @@ def write_header(self, buf):
     def write_body(self, buf):
         # ignore BOM, it confuses unrar
         if self.check_bom:
-            if [ord(b) for b in buf[:3]] == [239, 187, 191]:
+            if [b for b in buf[:3]] == [239, 187, 191]:
```

Comment:
Comparing a byte with a number does not make sense...

Reply:
Apparently, it only works in python 3. python 3 seems to convert single bytes to integers.